components/gc-dwnld/download_link-fr.html

```diff
@@ -0,0 +1,54 @@
+---
+{
+	"title": "Download link example",
```

Comment:
This title is used to generate the page title and the heading level 1 and because of that it need to be in French.

```suggestion
	"title": "Lien de téléchargement",
```